### PR TITLE
Add missing @phpstan- prefixed tags

### PR DESCRIPTION
**Pull request by zonuexe**
close https://github.com/phpstan/phpstan/issues/8697

### CODE REVIEW

File: src/Rules/PhpDoc/InvalidPHPStanDocTagRule.php [hunk 1]
```diff
@@ -23,11 +23,11 @@ class InvalidPHPStanDocTagRule implements Rule
 		'@phpstan-param',
 		'@phpstan-param-out',
 		'@phpstan-var',
-		'@phpstan-template',
```
Comment (zonuexe) [inline src/Rules/PhpDoc/InvalidPHPStanDocTagRule.php:26]: duplicated

File: tests/PHPStan/Rules/PhpDoc/InvalidPHPStanDocTagRuleTest.php [hunk 2]
```diff
@@ -33,6 +33,7 @@ public function testRule(): void
 				14,
 			],
 		]);
+		$this->analyse([__DIR__ . '/data/bug-8697.php'], []);
```
Comment (ondrejmirtes) [inline tests/PHPStan/Rules/PhpDoc/InvalidPHPStanDocTagRuleTest.php:40]: Please test it in a separate method. So that we always see all the failures.

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

**Comment by ondrejmirtes** [comment]
And FYI if you want this to be released much sooner, please base it on top of 1.9.x :)

**Comment by ondrejmirtes** [comment]
Thank you.

**Comment by ondrejmirtes** [comment]
How sooner? This sooner :D https://github.com/phpstan/phpstan/releases/tag/1.9.11

**Pull request merged**